typescript/looker/rtl/transport.ts

```diff
@@ -282,7 +282,7 @@ export type Authenticator = (props: any) => any
 /** Interface for API transport values */
 export interface ITransportSettings {
   [key:string] : any
-  /** base URL of host address */
+  /** base URL of API REST web service */
```

Comment:
This is a pretty big ripple effect throughout all SDKs if we change the behavior of base_url

Reply:
not changing the behavior of base url.  Just clarifying that this is NOT the looker instance url.  "host address" wasn't ambiguous when the only thing the SDK could talk to was the REST API service, but now that we have something that can talk to both, "host address" feels ambiguous.

Reply:
Ok. As long as it doesn't include `api/version` as part of base it is fine.